### PR TITLE
versioncontrol: add new version schema

### PR DESCRIPTION
**Pull request by mniewrzal**
What: Initial change to extend the version server with data necessary for binary auto-updater. 

Why: https://storjlabs.atlassian.net/browse/V3-2482

Please describe the tests:
 - Test 1: none
 
Please describe the performance impact: none

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

**Comment by mniewrzal** [comment]
@kaloyan-raev no tests yet, but I would like to combine version server with my updater tests https://github.com/storj/storj/pull/2970

File: versioncontrol/peer.go [hunk 1]
```diff
@@ -21,6 +21,8 @@ import (
 type Config struct {
 	Address  string `user:"true" help:"public address to listen on" default:":8080"`
 	Versions ServiceVersions
+
+	Version Versions
```
Comment (ethanadams) [inline versioncontrol/peer.go]: All the "versions" are confusing me :).  What if we change this to `Binary Versions` and then update the properties to something like:
```
# URL for specific binary
binary.bootstrap.minimum.url: ""

# peer version
binary.bootstrap.minimum.version: v0.0.1
```

Comment (mniewrzal) [reply versioncontrol/peer.go]: sounds good!

**Comment by mniewrzal** [comment]
> Looks good! Am I understanding correctly that you want to add tests for this in #2970?

Yes, but we can also add different tests in separate PR.

**Pull request merged**